crux-mir/README.md

```diff
@@ -56,12 +56,47 @@ versions corresponding to the last two `crux-mir` releases.
 [mir-json-readme]: https://github.com/GaloisInc/mir-json#readme
 
 
-## Setup: Build and Install
+## Setup
+
+### Build and Install
 
 Use GHC 9.6, 9.8, or 9.10. From the `crux-mir` directory, run:
 
     $ cabal install exe:crux-mir --overwrite-policy=always
 
+### Binary releases
+
+Both `crux-mir` and `mir-json` comes with nightly builds and stable binary releases. If you want to install the nightly build, do the following:
+
+* Go to the [crux-mir action](https://github.com/GaloisInc/crucible/actions/workflows/crux-mir-build.yml) page
+* Select the last build from `master` and scroll down the page, until you find and download the desired binary release (`crux-mir-macos...` or `crux-mir-ubuntu...`)
+* Unzip the downloaded archive and add it to your path
+* Do the same for `mir-json` by going to the [mir-json action](https://github.com/GaloisInc/mir-json/actions) page
+
+**Galois users only:** We provide a convenient install script for Linux and MacOS - you will need to provide a github access token to the `crucible` and `mir-json` repos:
+
+```
+$ ACCESS_TOKEN=xxx... install-crux.sh
+```
+
+### Library documentation
+
+The documentation for the `crucible` crate can be built locally by following these steps:
+
+* clone https://github.com/GaloisInc/mir-json.git 
+* build `mir-json` as described in the [Building from source](https://github.com/GaloisInc/mir-json?tab=readme-ov-file#building-from-source) section
+* build the docs with:
+  ```
+  rustdoc libs/crucible/lib.rs \
+    --edition 2021 \
+    --crate-name crucible \
+    --extern compiler_builtins=rlibs/libcompiler_builtins.rlib \
+    --extern core=rlibs/libcore.rlib \
+    -L rlibs \
+    --out-dir rustdocs \
+    --crate-type rlib
+  ```
+* open `rustdocs/crucible/index.html` in your browser!
```

Comment:
I wonder if this should go into the `mir-json` README instead, since nothing about this is specific to `crux-mir`. We could consider linking to that part of the `mir-json` README from the `crux-mir` README if you'd prefer.

Reply:
Done, see https://github.com/GaloisInc/mir-json/pull/124